.docs/contributing-book/src/code/bad_documentation/tests/harness.rs

```diff
@@ -0,0 +1,40 @@
+use fuels::{prelude::*, tx::ContractId};
+
+// Load abi from json
+abigen!(MyContract, "out/debug/bad_documentation-abi.json");
+
+async fn get_contract_instance() -> (MyContract, ContractId) {
+    // Launch a local network and deploy the contract
+    let mut wallets = launch_custom_provider_and_get_wallets(
+        WalletsConfig::new(
+            Some(1),             /* Single wallet */
+            Some(1),             /* Single coin (UTXO) */
+            Some(1_000_000_000), /* Amount per coin */
+        ),
+        None,
+    )
+    .await;
+    let wallet = wallets.pop().unwrap();
+
+    let id = Contract::deploy(
+        "./out/debug/bad_documentation.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "./out/debug/bad_documentation-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
+
+    let instance = MyContractBuilder::new(id.to_string(), wallet).build();
+
+    (instance, id.into())
+}
+
+#[tokio::test]
+async fn can_get_contract_id() {
+    let (_instance, _id) = get_contract_instance().await;
+
+    // Now you have an instance of your contract you can use to test each function
+}
```

Comment:
I have considered removing all tests since none are currently implemented however a counter point was to leave the harness in because otherwise it would need to be hand coded from scratch if it becomes relevant. Of course, we could simply bump the version and automatically generate a new one for us. I have chosen to leave this in for now in case someone wants to easily extract some test code and because it doesn't cost us anything to have it there especially since it's "hidden" from the public